apis/logging/v1beta1/collector_config_types.go

```diff
@@ -49,6 +49,11 @@ type RKE2Spec struct {
 	LogPath string `json:"logPath,omitempty"`
 }
 
+type KubeAuditLogsSpec struct {
+	Enabled bool   `json:"enabled,omitempty"`
+	LogPath string `json:"logPath,omitempty"`
```

Comment:
@alexandreLamarre @dbason, from my observation, you can only have 1 audit log file per node. 

To maintain "compatibility" with the rest of the CRD, I decided to use `LogPath`. However, I don't know if we should ask for the complete absolute path (`/path/audit.json`) instead of using the path + fileglob pattern (`/path/*.json`). What do you think?

Reply:
I can see it both ways, `fileglob` theoretically allows the user to come in with less working knowledge of Kubernetes audit logs, but comes with the risk of overmatching if improperly configured

Reply:
Following the Rancher Prime directive, I changed it to use something similar to what the [rancher-logging chart does](https://www.suse.com/support/kb/doc/?id=000021022), using `auditFilename` and `pathPrefix`.